tests/services/inventory/crawling_test.py

```diff
@@ -562,6 +560,7 @@ def _copy_file_from_gcs(file_path, *args, **kwargs):
             'folder': {'iam_policy': 3, 'resource': 3},
             'gsuite_group': {'resource': 4},
             'gsuite_group_member': {'resource': 1},
+            'gsuite_groups_settings': {'resource': 1},
```

Comment:
When setup correctly, there should be the same number of gsuite_groups_settings and gsuite_groups.